WinImageBuilder.psm1

```diff
@@ -670,6 +670,46 @@ function Compress-Image {
     Write-Output "MaaS image is ready and available at: $ImagePath"
 }
 
+function Start-Executable {

```

Comment:
This commit adds the possibility to create a password protected zip from the generated image.
This option is enabled by setting a value for the  "ZipPassword" parameter.